### PR TITLE
Update metrics supplementary guidelines to use the term Additive Property

### PR DESCRIPTION
**Pull request by utpilla**
## Changes
- Update metrics supplementary guidelines to use the term Additive Property in the decision criteria for choosing between Asynchronous Gauge and Asynchronous Counter

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: specification/metrics/supplementary-guidelines.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ Here is one way of choosing the correct instrument:
 * I want to **record** or **time** something, and the **statistics** about this
   thing are likely to be meaningful - use a [Histogram](./api.md#histogram).
 * I want to **measure** something (by reporting an absolute value):
-  * If it makes NO sense to add up the values across different sets of attributes, use
+  * If the measurement values DO NOT follow the [Additive property](#additive-property), use
```
Comment (MrAlias) [inline specification/metrics/supplementary-guidelines.md:75]: Using the additive property as an adjective as described in the sentence seems to make more sense here.

```suggestion
  * If the measurement are [non-additive](#additive-property), use
```

File: specification/metrics/supplementary-guidelines.md [hunk 2]
```diff
@@ -72,9 +72,9 @@ Here is one way of choosing the correct instrument:
...
     an [Asynchronous Gauge](./api.md#asynchronous-gauge).
-  * If it makes sense to add up the values across different sets of attributes:
+  * If the measurement values follow the [Additive property](#additive-property):
```
Comment (MrAlias) [inline specification/metrics/supplementary-guidelines.md:77]:
```suggestion
  * If the measurement are [additive](#additive-property):
```

**Comment by utpilla** [comment]
I plan to work on this PR and address the PR comments. Could one of the maintainers please reopen this PR?

File: specification/metrics/supplementary-guidelines.md (hunk 1 above)
Comment (utpilla) [reply specification/metrics/supplementary-guidelines.md]: I have updated the text.

File: specification/metrics/supplementary-guidelines.md (hunk 2 above)
Comment (utpilla) [reply specification/metrics/supplementary-guidelines.md]: I have updated the text.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**